cypress/integration/search_for_property.spec.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
+
+beforeEach(() => {
+  // Use properties fixture as mock stubbed data
+  cy.fixture('properties/properties.json').as('propertiesList')
+  cy.fixture('properties/property.json').as('property')
+  cy.server()
+})
+
+describe('Search for property', () => {
+  it('Search for property by postcode', () => {
+    cy.visit('http://localhost:5000/', { timeout: 3000 })
+
+    // Stub request for search on properties by address
+    cy.route('GET', 'api/v2/properties/?q=pitcairn', '@propertiesList')
```

Comment:
My IDE prodded me to say that `.route()` is deprecated in favour of [.intercept()](https://docs.cypress.io/api/commands/intercept.html#Comparison-to-cy-route) - do you think that would work for us? It looks like it might be a small change.

Reply:
Hey yes I'm going to address this in a next PR thanks for flagging!